src/output/gtk-shell.cpp

```diff
@@ -167,8 +167,21 @@ static void handle_gtk_shell_set_startup_id(wl_client *client, wl_resource *reso
  */
 static void handle_gtk_shell_system_bell(wl_client *client, wl_resource *resource,
     wl_resource *surface)
-{}
-
+{
+    if (surface)
```

Comment:
I think I also said it on the other PR, why not emit the signal on core, and jusst pass the wl_surface to it ... The plugin can then decide whether to ignore system bell on null surface or not.

Reply:
it does it now

Reply:
No, you didn't, just look at the code, it begins with an `if (surface)` ...

I mean you could just emit the code with the raw `wl_resource`. I can imagine you could get a system bell for a subsurface (which won't have a view), or a NULL system bell (which should still play sound afaik)